build/templates/UmbracoProject/appsettings.Development.json

```diff
@@ -0,0 +1,50 @@
+{
+  "Serilog": {
+    "MinimumLevel": {
+      "Default": "Information"
+    },
+    "WriteTo": [
+      {
+        "Name": "Async",
+        "Args": {
+          "configure": [
+            {
+              "Name": "Console"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  //#if (UsingUnattenedInstall)
+  "ConnectionStrings": {
+    "umbracoDbDSN": "CONNECTION_FROM_TEMPLATE"
+  },
+  //#endif
```

Comment:
Here is a section in the new appsettings I mentioned that will add in this extra setting if they have defined a value @elit0451